.github/workflows/lint.yml

```diff
@@ -31,3 +31,15 @@ jobs:
       - name: Running lint_whitespace_checks.sh on source files
         run: |
           ./build_tools/github_actions/lint_whitespace_checks.sh
+
+  llvm-integrate:
+    # This job can only be run on pull_request since GITHUB_BASE_REF is only set on PR.
+    if: "github.event_name == 'pull_request'"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checking out repository
+        uses: actions/checkout@v2
+
+      - name: Running lint_llvm_commit.sh on source files
+        run: |
+          ./build_tools/github_actions/lint_llvm_commit.sh ./build_tools/llvm_version.txt
```

Comment:
Sorry for going back and forth on this, but can we make this a step in the build_bazel.yml actually? Now that I'm seeing the "checks performed" it seems excessive that all jobs will have a separate check for llvm commit matching.

If this is a step of the bazel build, to first check that commits match, then build, we get the same outcome with a better organization I think. The script name can still be lint, and it will still give the same error.

Reply:
+1